modules/local/samtools_merge.nf

```diff
@@ -26,7 +26,7 @@ process SAMTOOLS_MERGE {
     def args2           = task.ext.args2   ?: ''
     def prefix          = task.ext.prefix ?: "${meta.id}"
     def reference       = fasta ? "--reference ${fasta}" : ""
-    def convert_to_cram = always_use_cram ? 
```

Comment:
@nvnieuwk , what's the point of this? The original module also supports cram.

Reply:
That was for when the input file is a BAM file. That argument could be specified to convert it to CRAM